blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,606 @@
+
+For example, focusing a dropdown will just say that you've focused a dropdown without saying what it's supposed to be for – rendering much of our UI totally unusable. With a buddy, it will read out the label on top of the element type. As an example, with buddies the OSD scale option will read approximately as "OSD Scale: Spin Box 100", but without, it will only read "Spin Box 100". There is no way to focus the labels either, so it's unlikely they can be read out without buddies.
+
+<Image src={require("./img/buddy.webp").default} />
+
+#### Tabstops
+
+Have you been in a situation where you have to use a keyboard to navigate your way through PCSX2 and found out that the ordering between options are complete nonsense? Tabstops are here to help!
+
+Tabstops are another property of Qt which allow an explicit ordering of elements when tabbing through a widget (i.e. focusing on one element then pressing Tab). Without them, the tab order is subject to the physical ordering of the elements in XML, which does not inherently correspond to the row/column order displayed and is highly unreliable and unrealistic to maintain.
+
+<Image src={require("./img/tabstops.webp").default} />
+
+<PCSX2PRLink authors="TellowKrinkle" prNums="12829,12830">
+  ### Improvements to PCSX2 related files handling on macOS
+</PCSX2PRLink>
+
+TellowKrinkle, our resident macOS housekeeper, has been working on adding file type associations for some of the file types that can be opened by PCSX2. This includes game dumps (ISO, CSO, ZSO, CHD, etc), save states, GS dumps, PNACH patches, and more.
+
+<PCSX2PRLink authors="TellowKrinkle" prNums="13407">
+  ### PCSX2 on macOS Now Uses Qt 6.10
+</PCSX2PRLink>
+
+For a long time, macOS has been stuck on Qt 6.7, due to Qt 6.10 dropping support for older macOS versions. It turns out Tellow discovered that it's trivial to patch compatibility for older macOS versions back into Qt 6.10. The rest is history!
+
+<PCSX2PRLink
+  authors="TheLastRar"
+  prNums="13466,12864,13360,13555,13557,13577,12864"
+>
+  ### Improved Display Surface (And Wayland!)
+</PCSX2PRLink>
+
+We have enabled Wayland support by default since PCSX2 2.4.0 release and in this cycle, AirGamer has yet again spearheaded the work to improve Wayland support even further. This time we're cutting out the cruft, resolving technical debt, and implementing it the way it was intended with Qt.
+
+For more technical details, previously the way we rendered into a "surface" was to render it to the widget directly. While this was the canonical method for older versions of Qt, the Qt maintainers mentioned that this was not recommended and a newer method was better suited. This has caused some issues, especially with Wayland.
+
+Instead, this new method now uses a Qt Window instead of a Qt Widget, avoiding issues inherited by the previous method and allowing for the removal of a number of Wayland specific workarounds. Which further improves Wayland compatibility with PCSX2.
```

Comment:
"Which further improves Wayland compatibility with PCSX2" 

Consider "This" instead of "Which"; "Which" implies something came before it, which with a period, the previous sentence is already stopped and the "Which" just kinda comes out of nowhere.